Dockerfile

```
@@ -20,7 +20,7 @@ RUN apk add --no-cache -U \
 
 RUN pip install mcstatus yq
 
-HEALTHCHECK CMD mcstatus localhost:$SERVER_PORT ping
+HEALTHCHECK --start-period=5m CMD mcstatus localhost:$SERVER_PORT ping
```

I know it conflicts with the PR title :) , but can you change this to 1m for the default:
```suggestion
HEALTHCHECK --start-period=1m CMD mcstatus localhost:$SERVER_PORT ping
```

---

Done.